Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:20.04

```

Comment:
Doing it this way would cause a really big docker container to be build. I would recommend using the ruby container as a base:

https://hub.docker.com/_/ruby

For example `2.7-alpine` would be a good base version that is really small.